docs/concepts/ale/ale-driver-development-concepts.md

```diff
@@ -0,0 +1,41 @@
+# ALE Driver Development Concepts
+
+## Drivers
+
+- While there are similarities between spacecraft, each instrument has particularities in:
+  - what data it collects/records
+  - how that data is shown
+
+- Drivers allow data from unique and differing spacecraft to be processed into a standard format.
+
+
+## Mixins
+
+- Inheritance structure in python
+  - details/explanation
+  - diagram
+  - link to more info
+
+- Why ALE uses mixins
+  - Mixing common behaviors of spacecraft
+    - Example from ALE
+
+
+## Ale::load
+
+- ? Not sure exactly what is intended for this section ?
```

Comment:
This needs to cover how loads works. That is, how it iterates over each driver and attempts to build an ISD with each. How loads quick rejects ISDs (using instrument IDs, look at the code and instructions for how to make an ale driver for examples). And how this can sometimes cause issues (sometimes a driver can succeed before the driver you want). 

Also mention isd_generate as an app that can do most of the boilerplate for you.

Reply:
> sometimes a driver can succeed before the driver you want

I mentioned this in my current draft, but I don't think I've seen this issue myself.  Would different `instrument_id`s prevent this from happening?  What can someone do if that happens to them?